python/cudf/cudf/tests/test_orc.py

```diff
@@ -1714,3 +1714,21 @@ def test_empty_columns():
 
     got_df = cudf.read_orc(buffer)
     assert_eq(expected, got_df)
+
+
+def test_orc_reader_zstd_compression(list_struct_buff):
+    expected = cudf.read_orc(list_struct_buff)
+    # save with ZSTD compression
+    buffer = BytesIO()
+    pyarrow_tbl = pyarrow.orc.ORCFile(list_struct_buff).read()
+    writer = pyarrow.orc.ORCWriter(buffer, compression="zstd")
+    writer.write(pyarrow_tbl)
+    writer.close()
+    try:
+        got = cudf.read_orc(buffer)
+        assert_eq(expected, got)
```

Comment:
@galipremsagar I had to pull the assert in, because of a failure when `read_orc` throws `"Unsupported compression type"`. `got` is not defined in that case so the test fails in the assert if it's placed after the whole `try` block.
I think we can avoid the issue by returning from the `pytest.mark.xfail(reason="nvcomp build doesn't have zstd")` branch.
Let me know if you have a preference.

Reply:
This is fine